HSDS/Common/Application.cpp

```diff
@@ -27,47 +27,53 @@ class Observer : public OpenDDS::DCPS::Observer {
     DDS::Topic_var topic = dw->get_topic();
     CORBA::String_var topic_name = topic->get_name();
     CORBA::String_var type_name = topic->get_type_name();
-    ACE_DEBUG((LM_INFO, "on_sample_sent topic=%C type=%C sample=%C\n", topic_name.in(), type_name.in(), OpenDDS::DCPS::to_json(s).c_str()));
+    ACE_DEBUG((LM_INFO, "INFO: Observer::on_sample_sent: topic=%C type=%C sample=%C\n",
```

Comment:
Add a configuration option for enabling/disabling the logging through the observer.